.github/workflows/build.yaml

```diff
@@ -0,0 +1,20 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    name: Build packages
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install formula
+        run: |
+          brew install -s Formula/snowcli.rb
+      - name: Check snow commands
+        run: |
+          snow --help
```

Comment:
Can we add a time check for the installation? So we know that it's below 5 min? I know it's not perfect but it would an early sign of degradation. WDYT?

Reply:
Nice idea, done